PropertySheets/Platform.props

```diff
@@ -25,7 +25,7 @@
 
   <ItemDefinitionGroup>
     <ClCompile>
-      <PreprocessorDefinitions>WIN32;_WIN32;_WINDOWS;WIN32_LEAN_AND_MEAN;%(PreprocessorDefinitions)</PreprocessorDefinitions>
+      <PreprocessorDefinitions>WIN32;_WIN32;_WINDOWS;%(PreprocessorDefinitions)</PreprocessorDefinitions>

```

Comment:
Version 3.0.2 needs it or won't compile.
That was not the cause of the errors.

This change builds out of the box (no tweaks) in all VS versions.